### ISSUE TITLE
Introduction and membership thread

### ISSUE THREAD

**Issue by ellaismer** [ellaism/meta#15]
If would like to help out Ellaism development and want to gain membership to this Github organization, post to this thread to introduce yourself. While you're there, remember to follow our simple [Github rules](https://github.com/ellaism/meta/blob/master/github.md).

**Comment by jorrinpollard** [comment]
Hi guys, I think this an interesting project and I'd like to help out with development. I can start with working on the myetherwallet to myellawallet conversion if help is still needed there.

**Comment by ellaismer** [comment]
@jorrinpollard Great and invited! For ellawallet, right now all functions work, but we have few icons/descriptions still refer to Ethereum rather than Ellaism. If you want, you can help to fix them.

In the mean time, we're trying to get Ledger support for ellawallet and other Ellaism wallets. If you happen to have time to help on that as well, just let me know!

**Comment by jorrinpollard** [comment]
Awesome! I'll start small and clean up the copy and icons for now. Once I'm done I'll look into Ledger support and dive in if it seems like something I can help with.

**Comment by cryptowd** [comment]
I never properly introduced myself.   I think I just wanted to see if I could make a contribution before worrying about anything official.

I've been interested in the Ellaism project and following it since the first weeks.  I'm new to blockchain technology in general with ELLA being the first project I bothered to set up a miner to support.  I've been surveying the whole crypto field, but I think this one could be the most promising and it's ideals align with what I think makes the blockchain so revolutionary. 

For my background, I currently wok as a freelance filmmaker for commercials and "branded content" for projects ranging from fortune 500 to government tourism.  However, in the past I have earned a living as a software engineer, system admin, and have spent a number of years in Hollywood climbing the ladder in the VFX industry.  I think I have some skills to bring, but my coding muscles are just a bit atrophied.

My plan is to continue working on MyEtherWallet.   I can keep rooting out Ethereum references where they pop up.  I also notice that My_Ether_Wallet has added some functionality that we don't have yet.  Cutting, pasting, and hacking code is probably my best way to get up to speed, so I may look into porting some of that.  But I'm open to anything, so feel free to reach out to me if you have something specific that needs attention.
